### ISSUE TITLE
✨ macOS Accessibility `Visible Character Range` data missing

### ISSUE THREAD

**Issue by godbout** [CodeEditApp/CodeEditTextView#114]
### Description

the `Visible Character Range` data is not provided by CodeEdit. the data is usually present in native apps. without the data it's hard to navigate through the TextArea with the AX Framework, for example in order to add Vim motions to CodeEdit through the AX APIs. 

### Alternatives Considered

one alternative of course is to ignore the fact that CodeEdit doesn't provide the `Visible Character Range`, but that would prevent third party apps to act properly on CodeEdit's TextArea.

### Additional Context

calling `AXUIElementCopyAttributeValue(axFocusedElement, kAXVisibleCharacterRangeAttribute as CFString, &axVisibleCharacterRange)` on the CodeEdit textarea returns `attributeUnsupported`. if the `kAXVisibleCharacterRangeAttribute` Attribute is properly provided, a call to the AX API will fill `axVisibleCharacterRange` with a `CFRange` containing the location of the first visible character (line), and the length until the last visible character (line).

### Screenshots

we can see that the data is also not provided through the Accessibility Inspector:

https://github.com/user-attachments/assets/fbb5839b-acb2-4994-b0c7-079e04df4fc2

**Comment by godbout** [comment]
> Was not aware of this parameter for accessibility. Moving this to the text view for tracking!

🙏

seeing the PR. fantastic. will test. thank you!
